modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -17,15 +17,15 @@ The *Pipelines* view in the *Developer* perspective lists all the pipelines in a
 .Procedure
 . In the *Pipelines* view of the *Developer* perspective, select a project from the *Project* drop-down list to see the Pipelines in that project.
 +
-.Pipelines view in the Developer perspective
-image::op-pipeline-list.png[]
+//Delete this image from the image library: image::op-pipeline-list.png[]
 +
 . Click the required Pipeline to see the *Pipeline Details* page. This page provides a visual representation of all the serial and parallel Tasks in the Pipeline. The Tasks are also listed at the lower right of the page. You can click the listed *Tasks* to view Task details.
 +
 //image::op-pipeline-details.png[Pipeline details]
 +
 . Optionally, in the *Pipeline Details* page:
 
+* Click the *Metrics* tab to see information about pipeline runs, such as pipeline success ratio, number of pipeline runs, pipeline run duration, and task run duration. You can use this information to improve the pipeline workflow and eliminate issues earlier in the pipeline lifecycle.
```

Comment:
Article missing "...number of the pipeline runs..."

Reply:
Adding Metrics page for reference
![image](https://user-images.githubusercontent.com/9964343/109954564-7f5d5c00-7d07-11eb-98ab-2166b1b15d29.png)

Reply:
@sounix000 Bullet listing the information in the Metrics page (the *Metrics* tab) will enhance the readability. Try and decide what works best and is appropriate. Thanks!

Reply:
@rishumehra - Changed those phrases to actual UI elements.
@Srivaralakshmi - Changed it to bullet points. Earlier I kept them in a single paragraph to reduce the length. 
@karthikjeeyar - I quoted the actual UI elements instead of generic phrases. We are not including images unless absolutely necessary, as it's a maintenance nightmare.